docs/reference/content/upgrading.md

```diff
@@ -95,19 +95,21 @@ the driver to use [Netty](http://netty.io/) instead.
 
 The following table specifies the compatibility of the MongoDB Java driver for use with a specific version of MongoDB.
 
-|Java Driver Version|MongoDB 2.6|MongoDB 3.0 |MongoDB 3.2|MongoDB 3.4|MongoDB 3.6|MongoDB 4.0|MongoDB 4.2|
```

Comment:
Slightly controversially I had to remove MongoDB 2.6 to get all the columns to fit